### PR TITLE
[fix] allow ALLOW_PRIVILEGED to be passed to kubelet and kube-api

### PR DESCRIPTION
**Pull request by jcrugzz**
This is something that we need for running docker in docker. Please let me know if you would consider this change. Thanks :)

### CODE REVIEW

**Comment by eparis** [comment]
ok to test

**Comment by eparis** [comment]
@MikeSpreitzer

File: cluster/ubuntu/util.sh [hunk 1]
```diff
@@ -245,6 +245,7 @@ KUBE_APISERVER_OPTS="\
  --admission-control=${2}\
  --service-node-port-range=${3}\
  --advertise-address=${4}\
+ --allow-privileged=${5}\
```
Comment (MikeSpreitzer) [inline cluster/ubuntu/util.sh]: Please add a function comment documenting this new argument.

**Comment by MikeSpreitzer** [comment]
Looks OK to me, except for the one missing comment.  But I have not tested it.

File: cluster/ubuntu/util.sh (hunk 1 above)
Comment (jcrugzz) [inline cluster/ubuntu/util.sh]: will do, thanks

**Comment by MikeSpreitzer** [comment]
Yes, the needed comment has been added.

**Comment by eparis** [comment]
ok to test

**Pull request merged**